docs/userguide.md

```diff
@@ -323,3 +323,40 @@ The argument(s) used is/are the list of context names of the additional clusters
 unjoined from this federation control plane. Clarifying, say the `host-cluster-context` used is
 `cluster1`, then on successful completion of the script used in example, both `cluster1` and `cluster2`
 will be unjoined from the deployed federation control plane.
+
+## Namespaced Federation
```

Comment:
Do we need to update the tile for above to highlight it is `Cluster Level Federation`?

Including title for `Automated Deployment` and `Manual Deployment`?

Reply:
I don't think I want to rewrite the other sections just yet.  Likely that will happen eventually, but initially the goal is making this feature available more than changing general perception of cluster-scoped federation.

Reply:
I think we do not need to rewrite the other sections, but just updating the title as follows should be good enough currently:

Automated Deployment -> Cluster Level Federation Automated Deployment
Manual Deployment -> Cluster Level Federation Manual Deployment